ratpack-rabbit-producer/src/test/groovy/smartthings/ratpack/rabbit/RabbitConnectionServiceSpec.groovy

```diff
@@ -13,6 +13,10 @@ class RabbitConnectionServiceSpec extends Specification {
 	@AutoCleanup
 	ExecHarness harness = ExecHarness.harness()
 
+	String getTestRabbitServer() {
+		return System.getenv("RABBIT_SERVER") ?: '192.168.99.100'
```

Comment:
I would prefer defaulting to localhost.  Given that docker on Linux and both RC for Mac and Windows bind to localhost it seems like a better default.  If thats not desirable for some then a custom domain name might be a good alternative.